apps/docs/public/content/docs/sharing-data/how-to-dataset/1-preparations/1-preparations.nn.mdx

```diff
@@ -25,25 +24,16 @@ Fyrst og fremst må du vite kva datasett du vil skildre, og informasjon som kan
 -   Om datasettet skal tilbydast ope
     -   og i så fall, om det er tilgjengeleg via filnedlasting eller API.
 
-<Alert
-    size='sm'
->
-    Tips: Spørsmål om korvidt datasettet skal tilbydast ope eller vere lukka, og kva for løyve som skal brukast, er
-    ofte mat for juristar. Mens de gjer desse avklaringane, kan du framleis publisere all den andre informasjonen. Når
-    de har blitt einige om korvidt datasettet skal vere ope, korleis det skal gjerast tilgjengeleg, og kva for løyve
-    som gjeld, kan du utvide skildringa med denne informasjonen.
+<Alert size='sm'>
+    Tips: Spørsmål om korvidt datasettet skal tilbydast ope eller vere lukka, og kva for løyve som skal brukast, er ofte
+    mat for juristar. Mens de gjer desse avklaringane, kan du framleis publisere all den andre informasjonen. Når de har
```

Comment:
Kan vi skrive noe annet en "mat for jurister"?
f.eks. "er ofte ei oppgåve for juristar"